addons/stock/models/stock_move.py

```diff
@@ -605,6 +606,13 @@ def _compute_reservation_date(self):
                     days = move.picking_type_id.reservation_days_before_priority
                 move.reservation_date = fields.Date.to_date(move.date) - timedelta(days=days)
 
+    @api.depends('forecast_availability', 'product_qty')
+    def _compute_will_be_fulfilled(self):
+        self.will_be_fulfilled = None
+        for move in self:
+            if move.product_uom.rounding:
```

Comment:
shouldn't this always be true since `uom.uom.rounding` is required and has a constraint to be >0 ?